dspace-spring-rest/src/test/java/org/dspace/app/rest/EPersonRestRepositoryIT.java

```diff
@@ -48,6 +48,8 @@
 
 public class EPersonRestRepositoryIT extends AbstractControllerIntegrationTest {
 
+    String defaultPassword = "7Testpass";
```

Comment:
there is already a password attribute in the https://github.com/DSpace/DSpace/blob/master/dspace-spring-rest/src/test/java/org/dspace/app/rest/test/AbstractIntegrationTestWithDatabase.java#L62